02_configure_host.sh

```diff
@@ -312,6 +312,14 @@ mkdir -p "${M3PATH}"
 detect_mismatch "${BMO_LOCAL_IMAGE:-}" "${BMOPATH}"
 clone_repo "${BMOREPO}" "${BMOBRANCH}" "${BMOPATH}" "${BMOCOMMIT}"
 
+# Clone a separate BMO main branch, to install deploy-cli.
```

Comment:
This needs to removed, when the BMO PR had merged.

Reply:
So we backport deploy-cli to all release branches?

Reply:
I was also thinking about it after I posted the comment. Bug since deploy-cli is argument compatible with deploy.sh, we can make dev-env call deploy.sh for other branches and deploy-cli for those that have it.

For example, symlinking deploy.sh to /usr/bin/deployc-cli if it doesn't exist, or vice-versa.

Reply:
That is one option too. @lentzi90 do you recall if we had serious blockers to that? I'm not fond of building too much around this, and IMO this is not something we want to backport.

This is also not called in so many places, that we couldn't just also if else it. Ansible can be made conditional as well. I feel like we need to keep testing with deploy.sh on the older branches, and use cli where it is naturally present.

Reply:
> I feel like we need to keep testing with deploy.sh on the older branches, and use cli where it is naturally present.

I agree. I would rather focus on IrSO than this deploy-cli. I guess we *can* publish the CLI binary, but if that makes people start using it instead of IrSO it would not be great.

Reply:
That sounds like a plan, although I would want to add that IrSO will not make this obsolete, as it still doesn't handle BMO.